### PR TITLE
fix(notifier): correct the bot token path

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

File: notifier/main.ts [hunk 1]
```diff
@@ -75,7 +75,10 @@ app.use(async (ctx, next) => {
   }
 });
 
-router.get(`/${bot.token}`, webhookCallback(bot, "oak"));
+router.get(
+  `/${bot.token.replace(/(\:)/g, "\\$1")}`,
```
Comment (KnorpelSenf) [inline notifier/main.ts:79]: Avoid regex.
```suggestion
  `/${bot.token.replaceAll(":", "\:")}`,
```

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]

File: notifier/main.ts (hunk 1 above)
Comment (roj1512) [reply notifier/main.ts:79]: Alright

File: notifier/main.ts [hunk 2]
```diff
@@ -75,7 +75,10 @@ app.use(async (ctx, next) => {
   }
 });
 
-router.get(`/${bot.token}`, webhookCallback(bot, "oak"));
+router.get(
+  `/${bot.token.replaceAll(":", "\:")}`,
```
Comment (KnorpelSenf) [inline notifier/main.ts:79]: Why not this?
```suggestion
  `/${bot.token.replaceAll(":", "\\:")}`,
```

Comment (roj1512) [reply notifier/main.ts]: Your previous review was really improper this time. But thanks for correcting it!

**Review by KnorpelSenf — APPROVED** [review]
LGTM

File: notifier/main.ts (hunk 2 above)
Comment (KnorpelSenf) [reply notifier/main.ts]: I edited immediately after commenting, but you were faster in applying it

Comment (roj1512) [reply notifier/main.ts]: Thanks, excuse my idiotism today!

**Pull request merged**